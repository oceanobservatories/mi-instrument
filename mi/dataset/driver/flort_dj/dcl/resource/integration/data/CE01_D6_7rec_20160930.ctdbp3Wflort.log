2016/09/30 17:17:44.459 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/30 17:30:03.223 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 17:30:26.644  10.1763,  3.66631,    0.984, 601, 219, 71, 30 Sep 2016 17:30:22
2016/09/30 17:31:01.836 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/30 18:30:03.554 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 18:30:26.979  10.3520,  3.68326,    1.027, 674, 195, 68, 30 Sep 2016 18:30:22
2016/09/30 18:31:02.180 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/30 19:28:04.764 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/30 19:30:02.915 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 19:30:26.340  10.2761,  3.67693,    1.090, 648, 252, 69, 30 Sep 2016 19:30:21
2016/09/30 19:31:01.541 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/30 20:28:07.283 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/30 20:30:03.114 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 20:30:26.535  10.2252,  3.67288,    1.032, 672, 258, 69, 30 Sep 2016 20:30:22
2016/09/30 20:31:01.732 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/30 21:28:06.597 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/30 21:30:03.387 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 21:30:26.813  10.4678,  3.69569,    1.016, 720, 189, 72, 30 Sep 2016 21:30:22
2016/09/30 21:31:02.006 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/30 22:28:06.634 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/30 22:30:03.121 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 22:30:26.537  10.4122,  3.69128,    0.952, 733, 242, 70, 30 Sep 2016 22:30:22
2016/09/30 22:31:01.743 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/30 23:28:06.214 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/30 23:30:04.599 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/30 23:30:28.024  10.3365,  3.68537,    0.953, 656, 257, 70, 30 Sep 2016 23:30:23
2016/09/30 23:31:03.214 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
