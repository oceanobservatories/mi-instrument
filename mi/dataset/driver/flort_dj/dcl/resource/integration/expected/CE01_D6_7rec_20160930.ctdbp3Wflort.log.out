
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684245426.6440000534 |          6 |         3684245422 | 3684245426.6440000534 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             601 |              71 |            219 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684249026.9790000916 |          6 |         3684249022 | 3684249026.9790000916 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             674 |              68 |            195 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684252626.3400001526 |          6 |         3684252621 | 3684252626.3400001526 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             648 |              69 |            252 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684256226.5349998474 |          6 |         3684256222 | 3684256226.5349998474 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             672 |              69 |            258 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684259826.8129997253 |          6 |         3684259822 | 3684259826.8129997253 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             720 |              72 |            189 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684263426.5369997025 |          6 |         3684263422 | 3684263426.5369997025 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             733 |              70 |            242 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3684182400 | telemetered | 3684267028.0240001678 |          6 |         3684267023 | 3684267028.0240001678 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             656 |              70 |            257 |        null

(7 rows)
